packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart

```diff
@@ -30,14 +32,23 @@ class TalkerScreenTheme {
     }
     return _defaultColors;
   }
+
+  factory TalkerScreenTheme.fromTheme(ThemeData theme, [LogColors? logColors]) {
+    return TalkerScreenTheme(
+      backgroundColor: theme.colorScheme.background,
+      textColor: theme.colorScheme.onBackground,
+      cardColor: theme.colorScheme.surface,
+    );
```

Comment:
Here we need to provide logColors argument

Reply:
fixed 👍